src/main/java/lotto/LottoApplicationController.java

```diff
@@ -0,0 +1,71 @@
+package lotto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoApplicationController {
+	private PurchaseAmount purchaseAmount;
+	private WinningNumbers winningNumbers;
+	private LottoStatistics lottoStatistics = new LottoStatistics();
+	private List<Lotto> lottos = new ArrayList<>();
+
+	public boolean enterPurchaseAmount() {
+		try {
+			purchaseAmount = new PurchaseAmount(InputView.enterPurchaseAmount());
+		} catch (IllegalArgumentException exception) {
+			OutputView.printMessage(exception.getMessage());
+			return true;
+		} catch (IllegalStateException exception) {
+			OutputView.printMessage(exception.getMessage());
+			return true;
+		}
+		return false;
+	}
+
+	public void printPurchaseQuantity() {
+		int purchaseQuantity = purchaseAmount.value() / CommonConstant.LOTTO_SALES_PRICE;
+		OutputView.printPurchaseQuantity(purchaseQuantity);
+
+		while (continuePurchase(purchaseQuantity)) {
+			lottos.add(new Lotto());
+			purchaseQuantity--;
+		}
+
+		OutputView.newLine();
+	}
+
+	private boolean continuePurchase(int purchaseQuantity) {
+		if (purchaseQuantity > CommonConstant.PURCHASE_FINISH) {
+			return true;
+		}
+		return false;
```

Comment:
코드를 인라인으로 사용해보는 건 어떨까요 ? :)
```suggestion
		return purchaseQuantity > CommonConstant.PURCHASE_FINISH;
```